src/xcode/fastlane/Fastfile

```diff
@@ -18,6 +18,8 @@
 ENV["LC_ALL"] = "de_DE.UTF-8"
 ENV["LANG"] = "de_DE.UTF-8"
 
+ENV["FASTLANE_SKIP_UPDATE_CHECK"] = "True"
```

Comment:
Do we get noticed ever when we have to update fastlane?

Reply:
Nobody cares on the CI and I run fastlane manually more than once per day. I'll get notice about updates and tbh, we don't have to get every dot update for fastlane.

Reply:
About the static SwiftLint: It's a hack until we have a new release that covers this issue in Xcode 12.5.